config/locales/en/transition_landing_page.yml

```diff
@@ -2,10 +2,12 @@ en:
   transition_landing_page:
     meta_title: The UK transition
     meta_description: The UK's transition period after Brexit comes to an end this year. Check the new rules from January 2021 and take action now.
-    page_header: The transition period ends in December
+    page_header: "The UK transition:<br/>time is running out"
     page_header_explainer: |
-      The UK has left the EU, and the transition period after Brexit comes to an end this year. Check the new rules from January 2021 and take action now.
-    take_action_title: Make sure you're ready
+      <span class="landing-page__intro-highlight">There are new rules</span> for businesses and citizens from 1 January 2021.
```

Comment:
Ideally, we would've had this content split into two locale items, so we don't need to hardcode markup in here (especially with styles attached) and we can use a better tag for semantics (such as `<mark>` or `<strong>`)

Reply:
Thanks, I'll tidy that up in a subsequent PR. 👍